custom_libs/subliminal_patch/providers/legendasdivx.py

```diff
@@ -325,9 +325,9 @@ def query(self, video, languages):
 
             search_url = _searchurl.format(
                     query=querytext,
-                    season=video.season,
-                    episode=video.episode,
-                    imdbid=video.series_imdb_id.replace('tt', '') if video.series_imdb_id else None,
+                    season='' if isinstance(video, Movie) else video.season,
+                    episode='' if isinstance(video, Movie) else video.episode,
+                    imdbid='' if isinstance(video, Movie) else video.series_imdb_id.replace('tt', '') if video.series_imdb_id else None,
```

Comment:
Why not `video.imdb_id.replace('tt', '')` instead of `''`?

Reply:
If imdb_id is available, it is already assigned to `querytext` here:
https://github.com/morpheus65535/bazarr/blob/e0d14a66e2d71a2d3f212e787cbeb2729c73946c/custom_libs/subliminal_patch/providers/legendasdivx.py#L297C1-L299C1
I kept `video.imdb_id.replace('tt', '')` in case of an episode.

I did a bit more digging in `legendasdivx.pt` site:
**For movies:**
I tried leaving the `querytext` empty and providing the imdb_id in the imdbid tag in the URL but the search results are not for the search item.
I also tested searching for the movie title in the `querytext` and setting the imdbid in the URL and it also does not work as expected.
It seems for movies, imdbid in the URL is either ignored or not working as expected (maybe a bug on the site).

**For episodes:**
Current URL is working properly, it takes into account the imdbid from the URL to provide subtitles just for that episode from that season.

Reply:
Ok lets keep it that way for now and we'll see how it goes in the future. Thanks!

Reply:
I was wrong for movies, `imdbid` in the URL was not working because by mistake I forgot the `id` in `imdbid`.
In resume, we can use the imdb id either in the `imdbid `or in `query`.

Reply:
Submit a quick PR and I'll merge it ASAP.

Reply:
The imdb search is much more optimized and should be prioritized if possible

Reply:
I agree, but that would need a bigger rework. 
This was my first contribution to this (or any other) project, so I didn't want to over-complicate.